### PR TITLE
uefi: Report if UX capsule was supported in the report metadata

### PR DESCRIPTION
**Pull request by superm1**
This is to support the action plan mentioned in https://github.com/rhboot/fwupdate/issues/66

### CODE REVIEW

File: plugins/uefi/fu-plugin-uefi.c [hunk 1]
```diff
@@ -645,6 +635,8 @@ fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 
 	/* for debugging problems later */
 	fu_plugin_uefi_test_secure_boot (plugin);
+	g_debug ("UX Capsule support : %s", data->ux_capsule);
+	fu_plugin_add_report_metadata (plugin, "UXCapsule", data->ux_capsule);
```
Comment (hughsie) [inline plugins/uefi/fu-plugin-uefi.c]: Probably needs a UEFI or Uefi prefix.

File: plugins/uefi/fu-plugin-uefi.c [hunk 2]
```diff
@@ -29,6 +29,10 @@
 #include "fu-plugin.h"
 #include "fu-plugin-vfuncs.h"
 
+struct FuPluginData {
+	const gchar	*ux_capsule;
```
Comment (hughsie) [inline plugins/uefi/fu-plugin-uefi.c]: I'd rather this was a gboolean, and then we can marshal it to a string when required.

File: plugins/uefi/fu-plugin-uefi.c (hunk 1 above)
Comment (superm1) [reply plugins/uefi/fu-plugin-uefi.c]: Adjusted

File: plugins/uefi/fu-plugin-uefi.c (hunk 2 above)
Comment (superm1) [reply plugins/uefi/fu-plugin-uefi.c]: Adjusted

**Pull request merged**